packages/react-components/react-drawer/src/components/Drawer/Drawer.test.tsx

```diff
@@ -2,25 +2,36 @@ import * as React from 'react';
 import { render } from '@testing-library/react';
 import { Drawer } from './Drawer';
 import { isConformant } from '../../testing/isConformant';
+import { resetIdsForTests } from '@fluentui/react-utilities';
+import { DrawerProps } from './Drawer.types';
 
 describe('Drawer', () => {
+  const testid = 'test';
+  const props = {
+    'data-testid': testid,
+  } as DrawerProps;
+
+  beforeEach(() => {
+    resetIdsForTests();
+  });
+
   isConformant({
     Component: Drawer,
     displayName: 'Drawer',
+    disabledTests: [
+      'component-handles-ref',
```

Comment:
why are we disabling so many things  ?

Reply:
Due to some factors:
- `component-handles-ref`: Drawer uses the Dialog under the hood and Dialog do not handle ref, as it is a renderless component
- `component-has-root-ref`: Same reason as above
- `component-handles-classname`: As Dialog is renderless, it won't have any classname. Drawer uses the DialogSurface component to render the classname. I will enforce the tests for this component later on, as soon as the implementation of `inline` drawer is done.
- `component-has-static-classnames-object`: Same as above
- `make-styles-overrides-win`: Same as above
- 'consistent-callback-args': Disabled that as the Drawer callback function uses the same signature as the Dialog. The Dialog have this tests disabled as well, so I had to force it here.

Reply:
Done!